_source/_posts/2018-08-28-nginx-auth-request.md

```diff
@@ -139,19 +139,15 @@ You'll need to download [Vouch](https://github.com/vouch/vouch-proxy) and compil
 
 Once you've got a binary, you'll need to create the config file to define the way you want Vouch to authenticate users.
 
-Copy `config/config.yml_example` to `config/config.yml` and read through the settings there. Most of the defaults will be fine, but you'll want to create your own JWT secret string and replace the placeholder value of `your_random_string`. 
+Copy `config/config.yml_example` to `config/config.yml` and read through the settings there. Most of the defaults will be fine, but you'll want to create your own JWT secret string and replace the placeholder value of `your_random_string`.
 
 The easiest way to configure Vouch is to have it allow any user that can authenticate at the OAuth server be allowed to access the backend. This works great if you're using a private OAuth server like Okta to manage your users. Go ahead and set `allowAllUsers: true` to enable this behavior, and comment out the `domains:` chunk.
 
-You'll need to choose an OAuth 2.0 provider to use to actually authenticate users. In this example we'll use Okta, since that's the easiest way to have a full OAuth/OpenID Connect server and be able to manage all your user accounts from a single dashboard. Before you can finish filling out the config file, you'll need to sign up for an Okta Developer account at [developer.okta.com/](https://developer.okta.com/). Once you create an account, click **Applications** in the top menu, and create a new application. Choose **Web** as the application platform.
+You'll need to choose an OAuth 2.0 provider to use to actually authenticate users. In this example we'll use Okta, since that's the easiest way to have a full OAuth/OpenID Connect server and be able to manage all your user accounts from a single dashboard.
 
-{% img blog/nginx-auth-request/okta-create-app.png alt:"Create a web application with Okta" width:"800" %}{: .center-image }
+{% include setup/cli.md type="web" %}
```

Comment:
```suggestion
{% include setup/cli.md type="web" loginRedirectUri="https://login.avocado.lol/auth" logoutRedirectUri="https://login.avocado.lol" %}
```

First redirect is so instructions match deleted screenshot. Logout is included because CLI prompts for one.